content/en/docs/Getting started/_index.md

```diff
@@ -13,82 +13,132 @@ description: >
 
     ```bash
     $ git remote remove origin
     $ git remote add origin https://github.com/my/example.git
     $ git push -u origin master
     ``` 
-  
+
 1. Get local copies of the project submodules so you can build and run your site locally:
 
     ```bash
     git submodule update --init --recursive
     ```
+1.  Switch to your `docsy-example` directory, and build your site:
+    
+    ```
+    hugo server
+    ```
+1.  Preview your site: http://localhost:1313/.
+    
 
-Alternatively, the following simple approach also works (though note this creates a link in GitHub between your project and the example site project):
+    {{% alert title="Tip" %}}
+If you've already cloned the example site with `--recurse-submodules --depth 1` and want to set a new origin repo, worry not! The following is one way to un-confuse Git and you can then proceed with the next step as normal. You can remove the `old` remote once you've pushed to your repo.
 
-1.  Fork the example site repo to your own GitHub account using the GitHub web UI by clicking **Fork** in [the example site's repo page](https://github.com/google/docsy-example). 
-1.  Rename your new fork appropriately by selecting **Settings** in its repo page and updating the **Repository name**.
+    $ git remote add old https://github.com/google/docsy-example.git
+    $ git fetch unshallow old
+
+    {{% /alert %}}
+
+### Using the GitHub UI
+
+Note that the following approach will create a link in GitHub between your project and the Docsy example site project):
+
+1.  In the [the Docsy example site's Github repo](https://github.com/google/docsy-example), click **Fork** and follow the prompts.
+1.  Rename your new fork:
+    1.  Click **Settings**, and type a new name in the **Repository name** field.
 1.  Make your own local working copy of your repo using `git clone`, recursing into the submodules:
 
     ```
-    $ git clone --recurse-submodules --depth 1 https://github.com/my/example.git
+    git clone --recurse-submodules --depth 1 https://github.com/my/example.git
     ```
 
-Once you have your local copy and GitHub repo, you can then edit away locally, pushing any changes you make to your GitHub repo using `git push`.
+You can now edit your local versions of the site's source files. To preview your site, run `hugo server`; by default, your site will be available at http://localhost:1313/. To push changes to your new repo, use `git push`.
+
+## Option 2: Use the Docsy theme in your own site
 
-### Use the Docsy theme in your own site
+Specify the [Docsy theme](https://github.com/google/docsy) like any other Hugo theme when creating or updating your site. This gives you all the theme-y goodness but you'll need to specify your own site structure.  You can either use the theme as a submodule (our recommended approach for easy updates), or just clone the theme into your project's `themes` subdirectory.
 
```

Comment:
Add here (as it applies to both approaches)

Whichever approach you use, for simplicity we recommend copying and editing our [example site configuration](#configuring-your-site) for your project, or you may get Hugo errors for missing parameters and values when you try to build your site.

Reply:
done.